internal/controller/datadogagent/global/utils.go

```diff
@@ -51,3 +54,24 @@ func setImageRegistry(manager feature.PodTemplateManagers, dda *v2alpha1.Datadog
 
 	return ""
 }
+
+func getInstallInfoValue() string {
+	toolVersion := "unknown"
+	if envVar := os.Getenv(InstallInfoToolVersion); envVar != "" {
+		toolVersion = envVar
+	}
+
+	return fmt.Sprintf(installInfoDataTmpl, toolVersion, version.Version)
+}
+
+func useCustomSeccomp(dda *v2alpha1.DatadogAgent) bool {
+	if componentOverride, ok := dda.Spec.Override[v2alpha1.NodeAgentComponentName]; ok {
+		if container, ok := componentOverride.Containers[apicommon.SystemProbeContainerName]; ok {
+			// Only ConfigMap is supported for now
```

Comment:
Don't we also support providing directly configData ? The operator will create a CM for it, but don't think it will be in the dda spec here ?

Reply:
I don't think we do anything with seccomp's configData in the code. I could only find custom seccomp data being used in [the overrides](https://github.com/DataDog/datadog-operator/blob/1aa8dc0fc9641dda6f0de46700a82d1746fe052b/internal/controller/datadogagent/override/container.go#L172) and I don't see any codepaths using `ConfigData`. I'll make a card for it in our backlog